### DISCUSSION TITLE
[Bug Report] openai.audio.translate  translated to other languages.

### DISCUSSION THREAD

**Discussion by sooni-pf** [openai/whisper#1604] [General]
I found out there are some bug when you use audio.translate function with prompt including foreign language.
The bug is that output is no more English. I want to assert this bug by sharing my experience.

I tried to translate youtube video (video id = bls_GjX-4U8, vBrQ3yzerMg) using audio.translate api with prompt.
the exact prompt that I used was : 
``` python
prompt=f"Insert punctuation marks and consider the title is {video_title}"
```
When {video_title} has korean language, the results were sometimes Korean, not English.

I just wanted to let developers know.
fyi. I am appreciated for the amazing whisper performance.

Thanks.

**Comment by phineas-pta** [comment]
your report should be at: https://community.openai.com/tag/whisper
